package.json

```diff
@@ -5,18 +5,25 @@
     "node": ">=14"
   },
   "scripts": {
-    "postversion": "git push --tags",
     "build:development": "cross-env RAILS_ENV=development NODE_ENV=development ./bin/webpack",
     "build:production": "cross-env RAILS_ENV=production NODE_ENV=production ./bin/webpack",
     "manage:translations": "node ./config/webpack/translationRunner.js",
+    "postversion": "git push --tags",
     "start": "node ./streaming/index.js",
-    "test": "${npm_execpath} run test:lint:js && ${npm_execpath} run test:jest",
-    "test:lint": "${npm_execpath} run test:lint:js && ${npm_execpath} run test:lint:sass",
-    "test:lint:js": "eslint --ext=.js,.jsx . --cache --report-unused-disable-directives",
-    "test:lint:sass": "stylelint \"**/*.{css,scss}\" && prettier --check \"**/*.{css,scss}\"",
+    "test": "yarn test:lint && yarn test:jest",
     "test:jest": "cross-env NODE_ENV=test jest",
-    "format": "prettier --write .",
-    "format-check": "prettier --check ."
+    "test:lint": "yarn test:lint:js && yarn test:lint:json && yarn test:lint:sass && yarn test:lint:yml",
+    "test:lint:fix": "yarn test:lint:js:fix && yarn test:lint:json:fix && yarn test:lint:sass:fix && yarn test:lint:yml:fix",
+    "test:lint:js": "eslint . --ext .js,.jsx --cache --report-unused-disable-directives",
+    "test:lint:js:fix": "yarn test:lint:js --fix",
```

Comment:
[IMO]
Creating a command that can easily perform autocorrection is a nice.
However, when executed with a wildcard like `test:*`, it executes both.

I suggest naming like `format:js`.